src/commerce.js

```diff
@@ -13,26 +13,28 @@ const defaultEventCallback = e => {
 };
 
 class Commerce {
-  constructor(publicKey, debug = false, config = {}) {
+  constructor(apiKey, debug = false, config = {}) {
     this.options = {
       version: 'v1',
       url: 'https://api.chec.io/',
       eventCallback: defaultEventCallback,
       disableStorage: false,
       cartLifetime: 30,
```

Comment:
```suggestion
      cartLifetime: 30,
      allowSecretKey: false,
```

Providing a default value for the config flag